base/abstractarraymath.jl

```diff
@@ -139,14 +139,25 @@ julia> b = [1 2; 3 4]
  1  2
  3  4
 
-julia> reverse(b, dims=2)
+julia> reverse(b, dims=1,2)
```

Comment:
I don't think this is what you mean. Did you run this example?

Reply:
Oh right, of course, thanks for spotting that. I hadn't been able to run the tests locally as I've been having trouble building on my machine (something to do with llvm), and due to something about the typing of keyword arguments I haven't been able to test anything out locally (though I found a workaround for that just now).  I was hoping it would work the first time on CI, but I'd been rather sloppy.